src/app/services/sms/sms.factory.ts

```diff
@@ -36,6 +70,10 @@ export const createSmsFactory = (
       sendVerificationOtp: () => {
```

Comment:
might be out of scope, but should we change this to Promise.reject too for consistency's sake?

Reply:
can be done together with https://github.com/opengovsg/FormSG/issues/941!